leapp/messaging/answerstore.py

```diff
@@ -7,6 +7,14 @@
 from leapp.utils.audit import create_audit_entry
 
 
+def _comment_out(key, text):
+    """Returns a commented-out string. If any newlines are present it properly deals with them"""
+    split_by_newline = [s for s in text.split('\n') if s.strip()]
+    res = '# {:<20}{}\n{}'.format('{}:'.format(key), split_by_newline[0],
```

Comment:
Hmm can't we just do:
```
first, rest = text.split('\n', 1)
if rest:
    rest = "# {}".format(rest.replace('\n', '\n# '))
...
```
Instead of looping over all of them?

Reply:
Ofc that requires a newline to be present

Reply:
I wanted to get rid of excessive newlines (like '\n\n\n') if they happen to appear.
But I'll give it some thinking, thanks.

Reply:
In the end I think it'll be more beneficial to have it like this https://github.com/oamg/leapp/pull/757/files#diff-3d756cc0c41cb26dd63ed05cfe760651cf1b33ec460fe203a5ae9aa8cdc0d7fdR319